index.html

```diff
@@ -61,7 +61,7 @@
 			// It initiates all other functionality, directly or indirectly
 			var UI = require('./js/rendererjs/uiManager.js');
 			// Need this for devtools to work
-			var Plugins = require('./js/rendererjs/pluginManager.js');
+			var Plugins;
```

Comment:
Oops, forgot to correct the index.html. That's what happens when you only grep inside a js folder, this would have broke all the things.